src/main/java/edu/unc/lib/boxc/migration/cdm/DestinationsCommand.java

```diff
@@ -132,6 +138,42 @@ public int add(@Mixin DestinationMappingOptions options) throws Exception {
         }
     }
 
+    @Command(name="map_archival_collections",
+            description = "Generate the destination mappings file by matching CDM field values to " +
+                    "the archival collection number in boxc")
+    public int generateArchival(@Mixin DestinationMappingOptions options) throws Exception {
```

Comment:
From a help text perspective, it seems like using the same options class for archival mapping generation and regular mapping generation might be problematic, since regular `generate` doesn't use the fieldName option, and archival doesn't use the cdmId field. The simplest way to address that could be to update the help text for the options to say which command they apply to. Alternatively, we could merge the two commands into one and make "--archival-collections" a flag to trigger that behavior instead. Or we could have two separate options classes, although there is a lot of overlap